scripts/publishedPackages.json

```diff
@@ -20,8 +20,6 @@
   "@adeira/monorepo-utils",
   "@adeira/murmur-hash",
   "@adeira/relay",
-  "@adeira/relay-runtime",
-  "@adeira/relay-utils",
```

Comment:
Maybe we should publish a new version with these packages where we update the readme and say that they are deprecated. Similar to [react-axe](https://www.npmjs.com/package/react-axe)

Reply:
I am going to deprecate them directly on NPM (so it's well-integrated and you should see it when installing). Additionally, these packages never reached a stable state so should not be a big deal.